descriptors/ModuleDescriptor-template.json

```diff
@@ -300,7 +300,8 @@
       "handlers": [
         {
           "methods": ["POST"],
-          "pathPattern": "/_/tenant"
+          "pathPattern": "/_/tenant",
+          "modulePermissions": ["finance-storage.transactions.collection.get"]
```

Comment:
Thinking about whether we need to add mod-finance-storage in 'required' of this module descriptor.
Recently we had these 3 stories: https://folio-org.atlassian.net/browse/MODINVOICE-532, https://folio-org.atlassian.net/browse/MODFIN-363, https://folio-org.atlassian.net/browse/MODORDERS-1053
to add missed required interfaces.
The call to finance-storage is made only during the migration process when the module is enabled, and not needed for regular business calls, but Eureka platform may be required to properly make this migration.
Thinking better to add, and we will remove after next release together with removing the migration class